README.md

```diff
@@ -580,6 +582,61 @@ The meta directive does not have any content and it accepts the following option
 * `points-to-pass`: module points to pass
 * `introduction`: module introduction as an HTML string
 
+Example module index.rst file:
+
+```rst
+Module 1 - Introduction
+=======================
+
+.. toctree::
+
+  introduction
+  topic
+
+.. aplusmeta::
+  :open-time: 2020-09-01 10:00
+  :close-time: 2020-09-30 14:00
+```
+
+Alternatively, one can also define these options in the conf.py file of the
+course in the following way.
+
+1. Include the aplusmeta directive in the `extensions` part of conf.py. Its
+   extension name is 'meta'.
+
+```python
+    extensions = [
+        ...
+        'meta',
+        ...
+    ]
+```
+
+2. Add the `aplusmeta_substitutions` variable in the conf.py file.
+
+```python
+    aplusmeta_substitutions = {
+        'open01': '2020-09-10 10:01'
+    }
+```
+
+This variable is a dictionary where keys are strings and values are dates
+in the usual format (see above). In the example above, you have defined a
+shortcut text "open01" for date "2020-09-10 10:01".
+
+3. Use the shortcut texts in the aplusmeta directive in a module index.rst:
+
+```rst
+    .. aplusmeta::
+      :open-time: open01
+      :close-time: 2019-09-20 12:00
```

Comment:
Not really part of this change, but I think it should be possible to write stuff like `2w` here to mark that the module should be open for 2 weeks. Or `${open02} + 2w` to say that it should close 2 weeks after `open02` variable.

Reply:
OK, but let's keep it out of this PR's scope.